packages/rpc/src/server.ts

```diff
@@ -38,6 +38,16 @@ JSON.stringify = function (
     return originalJsonStringify(value, wrapperReplacer, space)
 }
 
+declare module "fastify" {
```

Comment:
`fastify` is typed write?

Reply:
wdym?
it's type declarations for the field that we add via `fastify.decorateRequest()`